show/main.py

```diff
@@ -2424,5 +2425,7 @@ def tunnel():
 
     click.echo(tabulate(table, header))
 
+cli.add_command(kube.kubernetes)
```

Comment:
Suggest to add a comment above this line:

```python
# Add groups from other modules
```

Reply:
done